src/index.html

```diff
@@ -5,8 +5,412 @@
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="logo">
+        <img src="./images/logo.png" alt="logo">
+      </div>
+      <nav class="menu">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#">
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Смартфоны и телефоны
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              data-qa="nav-hover"
+              href="#">
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Гаджеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Планшеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Фото
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link nav__video" href="#">
+              Видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="catalog">
+        <section class="card" data-qa="card">
+          <div class="card__photo">
+            <img
+              class="card__photo-img"
+              src="images/imac.jpeg"
+              alt=""
+            >
+          </div>
+          <div class="card__content">
+            <div class="card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</div>
+            <div class="card__code">Код товара: 195434</div>
+            <div class="card__rate">
+              <div class="card__stars">
+                <div class="card__star">
+                  <img src="images/star-active.svg" alt="star active">
+                </div>
+                <div class="card__star">
+                  <img src="images/star-active.svg" alt="star active">
+                </div>
+                <div class="card__star">
+                  <img src="images/star-active.svg" alt="star active">
+                </div>
+                <div class="card__star">
+                  <img src="images/star-active.svg" alt="star active">
+                </div>
+                <div class="card__star">
+                  <img src="images/star.svg" alt="star">
+                </div>
+              </div>
+              <div class="card__rewiew">Отзывов: 5</div>
+            </div>
+            <div class="card__price">
+              <div class="card__price-info">Цена:</div>
+              <div class="card__price-amount">69 999 грн</div>
```

Comment:
```suggestion
            <div class="price card__price">
              <div class="price__info">Цена:</div>
              <div class="price__amount">69 999 грн</div>
```
Так виглядало б краще